README.md

```diff
@@ -55,6 +55,8 @@ releases of the toolkit.
 * Optional, but recommended:
     * Gradle 3.1 or greater, needed for building the GATK. We recommend using the `./gradlew` script which will
       download and use an appropriate gradle version automatically (see examples below).
+    * [git-lfs](https://git-lfs.github.com/) 1.1.0 or greater, needed for building the GATK. Used to download the large files required to run the build and to run the test suite.
+      Run `git lfs install` after downloading, followed by `git lfs pull` from the root of your git clone to download the large files. The download is several hundred megabytes.
```

Comment:
Maybe we should split into "build" and "runtime" requirements?

Reply:
Also, can you check whether `git lfs pull` still works?

Reply:
I split this into separate sections for runtime and build. Updated to state that the lfs download is around 2GB. And `git lfs pull` does work.